.github/workflows/tests.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         echo "$HOME/.local/bin" >> $GITHUB_PATH
         echo "$HOME/bin" >> $GITHUB_PATH
         echo "PYTHONPATH=$PWD" >> $GITHUB_ENV
+        echo "PYTEST_ADDOPTS=--basetemp=build" >> $GITHUB_ENV
     - name: Test
       run: |
-        python3 -m pytest -n $(nproc) -vv
+        python3 -m pytest -n $(nproc) -vv -s
```

Comment:
Is the output relevant here?

Reply:
The output makes it easier to debug problems. As one usually only looks into the log, if there is a problem (either a failing test or an unexpected test result), it makes sense to me to keep it.

Reply:
I have changed my mind and removed the option. In the most cases (i.e. failing tests) pytest will return the output anyway.